packages/design-mui/src/components/MuiProvider/MuiProvider.tsx

```diff
@@ -3,41 +3,24 @@ import type { XProviderProps } from "@hatchifyjs/react-ui"
 import {
   HatchifyPresentationProvider,
   HatchifyPresentationDefaultValueComponents,
-  HatchifyPresentationDefaultFieldComponents,
 } from "@hatchifyjs/react-ui"
 import { MuiLayout } from "../MuiLayout"
-import { MuiForm } from "../MuiForm"
-import { MuiDetails } from "../MuiDetails"
 import MuiDataGrid from "../MuiDataGrid/MuiDataGrid"
-import {
-  String as StringInput,
-  Number as NumberInput,
-  Date as DateInput,
-  Boolean as BooleanInput,
-  Relationship as RelationshipInput,
-} from "./DefaultFieldComponents"
 import { Relationship, RelationshipList } from "./DefaultDisplayComponents"
 
 export const MuiProvider: React.FC<XProviderProps<Theme>> = ({ children }) => {
   return (
     <HatchifyPresentationProvider
       Collection={MuiDataGrid}
       Layout={MuiLayout}
-      Details={MuiDetails}
-      Form={MuiForm}
+      // Details
+      // Form
```

Comment:
Should we take these comments out completely?

Reply:
idk i was just leaving them as a reference for future work, i'm indifferent as to whether to keep them or not 😅